lib/lib.es6.d.ts

```diff
@@ -21042,6 +21043,7 @@ type ConstrainDOMString = string | string[] | ConstrainDOMStringParameters;
 type ConstrainDouble = number | ConstrainDoubleRange;
 type ConstrainLong = number | ConstrainLongRange;
 type CryptoOperationData = ArrayBufferView;
+type DOMExceptionType = "IndexSizeError" | "DOMStringSizeError" | "HierarchyRequestError" | "WrongDocumentError" | "InvalidCharacterError" | "NoDataAllowedError" | "NoModificationAllowedError" | "NotFoundError" | "NotSupportedError" | "InUseAttributeError" | "InvalidStateError" | "SyntaxError" | "InvalidModificationError" | "NamespaceError" | "InvalidAccessError" | "ValidationError" | "TypeMismatchError" | "SecurityError" | "NetworkError" | "AbortError" | "URLMismatchError" | "QuotaExceededError" | "TimeoutError" | "InvalidNodeTypeError" | "DataCloneError" | "EncodingError" | "NotReadableError" | "UnknownError" | "ConstraintError" | "DataError" | "TransactionInactiveError" | "ReadOnlyError" | "VersionError" | "OperationError" | "NotAllowedError";
```

Comment:
I don't see "DOMStringSizeError" in https://developer.mozilla.org/en-US/docs/Web/API/DOMException.....